### PR TITLE
PPT-448: human medicines plastic weight page impl

### PR DESCRIPTION
**Pull request by tombart**
Screenshot 2021-03-22 at 14 00 02

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/returns/forms/HumanMedicinesPlasticWeightSpec.scala [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      "provided with empty data" in {
+
+        val input = Map.empty[String, String]
+        val expectedErrors =
+          Seq(FormError(totalKg, weightEmptyError))
+
+        testFailedValidationErrors(input, expectedErrors)
+      }
+
+      "contains alphanumerical or special character" in {
```
Comment (tmc-ee) [inline test/uk/gov/hmrc/plasticpackagingtax/returns/forms/HumanMedicinesPlasticWeightSpec.scala:83]: Am I right to assume that decimals are covered by this test too?

**Review by tmc-ee — APPROVED** [review]

File: test/uk/gov/hmrc/plasticpackagingtax/returns/forms/HumanMedicinesPlasticWeightSpec.scala (hunk 1 above)
Comment (tombart) [reply test/uk/gov/hmrc/plasticpackagingtax/returns/forms/HumanMedicinesPlasticWeightSpec.scala:83]: yeap, all non numeric chars should be invalid, I'll add decimals to test scenario

File: app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/HumanMedicinesPlasticWeight.scala [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.returns.models.domain
+
+import play.api.libs.json.{Json, OFormat}
+
+case class HumanMedicinesPlasticWeight(totalKg: Option[Long] = None)
```
Comment (PauloPortugal) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/HumanMedicinesPlasticWeight.scala:21]: Yes this is the thing I don't like. But I am addressing on my ticket. Happy to continue for now, but if something is mandatory we should not mark it as optional.

I will discuss with you later :)

**Review by PauloPortugal — APPROVED** [review]
LGTM.
We should discuss the Optional type in mandatory form fields.

File: app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/HumanMedicinesPlasticWeight.scala (hunk 2 above)
Comment (tombart) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/HumanMedicinesPlasticWeight.scala:21]: ok, cool, happy to discuss, I'll merge that for now, and we can catch up tomorrow on the way forward

**Pull request merged**